### PR TITLE
Remove superuser-related docs for supabase

### PR DESCRIPTION
**Pull request by justinvdm**
On Nov 5, users will not be able to be superusers on supabase anymore: https://github.com/supabase/supabase/discussions/9314.

We should merge this docs change _on Nov 5_.

### CODE REVIEW

File: docs/06-tutorials/supabase-clone-environments.md [hunk 1]
```diff
@@ -31,44 +32,41 @@ You can follow along in the video below as our founder, Peter Pistorius takes yo
...
 ![Snaplet onboarding connect your database](../../static/screenshots/supabase-integration/onboarding_connect_db.png)
 
 You’ll then want to enter the credentials of your production Supabase project. Find the "Connection string" in Supabase via `Organization > Project > Settings > Database > Connection string` (At the bottom of the page.)
 
 Remember that the password is the same password you used when creating the Supabase project.
 
-
 ![Supabase console connection info](../../static/screenshots/supabase-integration/supabase_connection_info.png)
 
 You’ll then have to confirm providing Snaplet access to your database. Snaplet will prompt you to only provide `read-only` access to your database. We have a guide in our documentation on how to go about this and you can check that out [here](/guides/postgresql#create-a-read-only-role).
```
Comment (jgoux) [inline docs/06-tutorials/supabase-clone-environments.md]: We need to update this guide to add the `BYPASSRLS` role

File: docs/06-tutorials/supabase-clone-environments.md [hunk 2]
```diff
@@ -78,23 +76,17 @@ If you don’t already have a developer database setup on Supabase, you’ll nee
 3. Enter your project details
 4. Wait for the new database to launch
 
-> Remember the password you use when creating the project. You’ll need this password to connect your database to Snaplet later. 
-
-### Step 5: Make your `postgres` user a superuser
```
Comment (jgoux) [inline docs/06-tutorials/supabase-clone-environments.md:83]: Should we say that for restoring, they **must** use the `postgres` role?

Also now that this step is deleted we should decrement the number of the next steps to 5 and 6. 👍

**Review by jgoux — CHANGES_REQUESTED** [review]
Formatting details and a question about the `postgres` role. :)

File: docs/06-tutorials/supabase-clone-environments.md (hunk 1 above)
Comment (justinvdm) [reply docs/06-tutorials/supabase-clone-environments.md]: That'd only be needed once we make that change to the introspection queries though, right? Or do you just mean we should do this ahead of time?

File: docs/06-tutorials/supabase-clone-environments.md (hunk 2 above)
Comment (justinvdm) [reply docs/06-tutorials/supabase-clone-environments.md:83]: :+1: and :+1:, thank you!

File: docs/06-tutorials/supabase-clone-environments.md (hunk 1 above)
Comment (jgoux) [reply docs/06-tutorials/supabase-clone-environments.md]: It should always be applied, right now if a user uses RLS rules we won't bring the matching rows into the snapshots.

**Comment by justinvdm** [comment]
@jgoux thanks for the review! Could you take another look when you have a moment?

**Review by jgoux — APPROVED** [review]
Looks good to me, did you try executing the `BYPASSRLS` statement?

Good job!

**Comment by justinvdm** [comment]
> Looks good to me, did you try executing the `BYPASSRLS` statement?
> 
> Good job!

Looks like it worked :) only via psql, not via the SQL editor, I'll note that in the docs

**Comment by justinvdm** [comment]
Thanks for the review @jgoux :) I'll keep this open until November 5, at which point we should merge. Heads up on this @peterp.

**Comment by justinvdm** [comment]
On second thought, merging - people have had a month already to migrate. We might just need to tell people to migrate.

**Pull request merged**